### ISSUE TITLE
Option to delete "rescheduled" message

### ISSUE THREAD

**Issue by Gameniti** [marcoancona/MineTime#236]
tl;dr: From a business perspective, it is nice to see that an event has been rescheduled. But from a personal-use perspective, I think it would be nice to delete/disable/hide the number of rescheduled times on specific events. (I think Hide would be the best option because, in case of having an event where you need to know afterward how often it has been rescheduled, it would be helpful to know)

-----

As an example:
I work in the event industry where events get rescheduled quite often, caused by various things. In this case, it is nice to know that events have been rescheduled multiple times (Maybe for that I would prefer an option to add notes why it was rescheduled). But even then I don't need to know every event that has been rescheduled.

For my account at home, I don't need that option enabled all the time. Some events may be nice to know (this is why hiding would be better than disabling) but most of the time I find it annoying that it says the event has been rescheduled X times. For example, I just changed the start of a birthday to be 20 minutes earlier (to get the correct notifications) than I initially thought, but it says the event has been rescheduled once before, which true, but not really necessary (because the event of itself didn't change, just the personal timing for myself)
